### PR TITLE
Timeout

### PR DESCRIPTION
**Pull request by mamcderm-usgs**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.6.14] - 2018-12-07
+### Updated
+-- [mamcderm@usgs.gov] - increase timeouts
```
Comment (isuftin) [inline CHANGELOG.md]: You also added the `ignore_readdir_race` flag into the command set so please add that under an `### Added` header

File: recipes/aide.rb [hunk 2]
```diff
@@ -83,6 +83,7 @@
 execute 'init_aide' do
   user 'root'
   command '/usr/sbin/aide --init'
+  timeout 7200
```
Comment (isuftin) [inline recipes/aide.rb]: For every place you've added this value, can you parameterize this? Something like `default['stig']['aide']['timeout'] = 7200` and then use that value here as in

`timeout node['stig']['aide']['timeout']`

This makes the cookbook more dynamic and doesn't force us to re-release if we (or another user) finds that this value is not correct for their situation

File: recipes/audits.rb [hunk 3]
```diff
@@ -25,8 +25,9 @@
 
 bash 'find user and group orphaned files and directories' do
   user 'root'
-  code "df --local -P | awk {'if (NR!=1) print $6'} | uniq | xargs -I '{}' find '{}' -xdev -nouser -nogroup | while read fn;do chown root:root \"$fn\";done"
-  only_if "test -n \"$(df --local -P | awk {'if (NR!=1) print $6'} | uniq | xargs -I '{}' find '{}' -xdev -nouser -nogroup)\"", user: 'root'
+  timeout 7200
```
Comment (mamcderm-usgs) [inline recipes/audits.rb]: @isuftin - should this timeout, and the one on the guard interpreter also have a node parameter or just the aide timeout?

Comment (isuftin) [reply recipes/audits.rb]: I'd say anywhere you're applying a hard-coded timeout, let's parametrize that

**Comment by isuftin** [comment]
Looking good

File: recipes/audits.rb (hunk 3 above)
Comment (mamcderm-usgs) [reply recipes/audits.rb]: Good point, I setup two parameters.  Let me know if you see any other changes needed, otherwise, will merge and tag.

**Comment by mamcderm-usgs** [comment]
ok, I will go ahead and merge.

**Pull request merged**